lib/client.js

```diff
@@ -108,9 +108,10 @@ Client.prototype.getRaw = function(url) {
   });
 };
 
-Client.prototype.authGet = function(_path, _accessToken) {
+Client.prototype.authGet = function(_path, _accessToken, _skipPrefix) {
+  _skipPrefix = (typeof _skipPrefix === 'undefined') ? true : _skipPrefix;
   return yarp({
-    url: this.buildUrl('auth', _path, { access_token: _accessToken }, true)
+    url: this.buildUrl('auth', _path, { access_token: _accessToken }, _skipPrefix)
```

Comment:
Instead of reassigning `_skipPrefix` just do it in one line:
`this.buildUrl('auth', _path, { access_token: _accessToken }, _skipPrefix || true)`